src/components/ApiDocs/ApiComment.tsx

```diff
@@ -15,6 +16,23 @@ export const ApiComment = ({ apiComment, codeBlock }: ApiCommentProps) => {
   }
   const commentList = apiComment.map((snippet, idx) => {
     if (snippet.kind === 'code') {
+      if (snippet.text.indexOf('```') !== -1) {
+        // If the comment contains a code block extract the language and render an MDXCode component
```

Comment:
can you give an example of this?

Reply:
This is the current use case 
```
{
      "kind": "code",
      "text": "```ts\nimport { DataStore, syncExpression } from 'aws-amplify/datastore';\nimport { Post, Comment } from './models';\n\n\nDataStore.configure({\n\tsyncExpressions: [\n\t\tsyncExpression(Post, () => {\n\t\t\treturn (post) => post.rating.gt(5);\n\t\t}),\n\t\tsyncExpression(Comment, () => {\n\t\t\treturn (comment) => comment.status.eq('active');\n\t\t})\n\t]\n});\n```"
}
```
The comment node is marked as code but the text inside the comment could be either a code block or just a singular code tag, we are using ``` to differentiate the code blocks from the ` code tags